src/platform/Ameba/ConfigurationManagerImpl.cpp

```diff
@@ -36,9 +36,25 @@ namespace DeviceLayer {
 
 using namespace ::chip::DeviceLayer::Internal;
 
-/** Singleton instance of the ConfigurationManager implementation object.
+/** Singleton pointer to the ConfigurationManager implementation.
  */
-ConfigurationManagerImpl ConfigurationManagerImpl::sInstance;
+ConfigurationManager * gInstance;
```

Comment:
consider setting this explicitly to nullptr.

Previous code may have assumed that ConfigurationMgr *never* can fail. Would it be worth it to initialize this with GetDefaultInstance()?

Reply:
I've set it explicitly to nullptr.